packages/scaffolding-cli/package.json

```diff
@@ -23,28 +26,44 @@
     "fs-extra": "^9.0.0",
     "lodash": "^4.17.15",
     "prompts": "^2.3.1",
-    "replace-in-file": "^5.0.2",
-    "simple-git": "^2.5.0",
+    "replace-in-file": "^6.0.0",
+    "simple-git": "^1.132.0",
```

Comment:
did we downgrade simple-git?

Reply:
Yeah good point, let me see if it's because of a dependable change.

Reply:
AB#1979